src/components/pages/JoinPage/index.jsx

```diff
@@ -63,7 +66,9 @@ class JoinPage extends React.Component {
 
   checkEmail = async (e) => {
     e.preventDefault();
+
     let rtn = await this.props.checkEmail(this.state.email.trim());
```

Comment:
@Aichnerc do you see it? I thought you are a `const` nazi <3.

@Theralun When you adjust this you should have a look at the complete code. There are more shadow constants.